lib/ts/recipe/thirdpartyemailpassword/recipeImplementation.ts

```diff
@@ -0,0 +1,68 @@
+import { RecipeInterface } from "supertokens-web-js/recipe/thirdpartyemailpassword";
+import { NormalisedConfig } from "./types";
+
+export default function getRecipeImplementation(
+    webJsRecipe: RecipeInterface,
+    authReactConfig: NormalisedConfig
+): RecipeInterface {
+    return {
+        submitNewPassword: async function (input) {
+            return webJsRecipe.submitNewPassword(input);
+        },
+        sendPasswordResetEmail: async function (input) {
+            return webJsRecipe.sendPasswordResetEmail(input);
+        },
+        doesEmailExist: async function (input) {
+            return webJsRecipe.doesEmailExist(input);
+        },
+        getResetPasswordTokenFromURL: function (input) {
+            return webJsRecipe.getResetPasswordTokenFromURL(input);
+        },
+        getAuthorisationURLFromBackend: async function (input) {
+            return webJsRecipe.getAuthorisationURLFromBackend(input);
+        },
+        getAuthorizationURLWithQueryParamsAndSetState: async function (input) {
+            return webJsRecipe.getAuthorizationURLWithQueryParamsAndSetState(input);
+        },
+        emailPasswordSignIn: async function (input) {
+            return webJsRecipe.emailPasswordSignIn(input);
+        },
+        emailPasswordSignUp: async function (input) {
+            return webJsRecipe.emailPasswordSignUp(input);
```

Comment:
Since we refactored tpep to use the auth react versions of emailpassword and thirdparty the onHandleEvent does not need to be called manually inside tpep implementation. Removed it from thirdpartysigninup as well